src/platform/plugins/shared/data/server/search/session/session_service.ts

```diff
@@ -191,7 +196,10 @@ export class SearchSessionService implements ISearchSessionService {
     const sessionStatuses = await Promise.all(
       findResponse.saved_objects.map(async (so) => {
         const sessionStatus = await getSessionStatus(
-          { internalClient: internalElasticsearchClient },
+          {
+            internalClient: internalElasticsearchClient,
+            asUserClient: asCurrentUserElasticsearchClient,
+          },
```

Comment:
I think we can actually just replace the `internalClient` with the `asUserClient` and use it for the async search status calls as well... The internal client was originally used because there was a scheduled task that had to get the search session status, but this is no longer the case.